From d05fd7ff7b5f9a64df4977842de1f338d22d55bd Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Fri, 11 May 2018 13:55:07 +0200
Subject: [PATCH] Update sdhci-msm.c

---
 drivers/mmc/host/sdhci-msm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/host/sdhci-msm.c b/drivers/mmc/host/sdhci-msm.c
index 92c483ec6..eb1f6853b 100644
--- a/drivers/mmc/host/sdhci-msm.c
+++ b/drivers/mmc/host/sdhci-msm.c
@@ -1151,7 +1151,7 @@ static int sdhci_msm_probe(struct platform_device *pdev)
 	msm_host->bus_clk = devm_clk_get(&pdev->dev, "bus");
 	if (!IS_ERR(msm_host->bus_clk)) {
 		/* Vote for max. clk rate for max. performance */
-		ret = clk_set_rate(msm_host->bus_clk, INT_MAX);
+		ret = clk_set_rate(msm_host->bus_clk, 192000000);
 		if (ret)
 			goto pltfm_free;
 		ret = clk_prepare_enable(msm_host->bus_clk);
@@ -1190,7 +1190,7 @@ static int sdhci_msm_probe(struct platform_device *pdev)
 	}
 
 	/* Vote for maximum clock rate for maximum performance */
-	ret = clk_set_rate(msm_host->clk, INT_MAX);
+	ret = clk_set_rate(msm_host->clk, 192000000);
 	if (ret)
 		dev_warn(&pdev->dev, "core clock boost failed\n");
 
